spec/unit/git_secrets_spec.rb

```diff
@@ -21,16 +22,16 @@
   it 'installs git-secrets hooks for all users' do
     global_hooks_dir = '/usr/local/share/githooks'
     hooks = [
-      'pre-commit',
-      'commit-msg',
-      'prepare-commit-msg'
+      OpenStruct.new(name: 'pre-commit', git_secrets_hook: 'pre_commit_hook'),
+      OpenStruct.new(name: 'commit-msg', git_secrets_hook: 'commit_msg_hook'),
+      OpenStruct.new(name: 'prepare-commit-msg', git_secrets_hook: 'prepare_commit_msg_hook')
     ]
```

Comment:
I'm wondering if this would be clearer with a hash, avoiding the `ostruct` dependency?

```ruby
hook_data = {
  hook_directory: 'pre-commit', hook_name: 'pre_commit_hook',
  # ... 
}
```